### PR TITLE
Mirror roslyn,visualfsharp,msbuild branches

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

**Comment by mmitche** [comment]
@dagood I can't remember whether the wildcard without a / actually works....(e.g. to cover devfoobar and devbaz)

**Comment by dagood** [comment]
Interesting... so Maestro needs the full `/**/*`:

https://github.com/dotnet/versions/blob/82ad5fa276bcec84548ba7e31d747be0164f99f5/Maestro/src/Microsoft.DotNet.Maestro.WebApi/Models/SubscriptionsModel.cs#L89-L93

But since it uses `- 5`, something like `master/**/*` matches anything starting with `master`. So you can make it work but it'll look weird.

**Comment by mmitche** [comment]
Perfect.

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -793,14 +793,21 @@
         "https://github.com/dotnet/diagnostics/blob/master/**/*",
         "https://github.com/dotnet/machinelearning/blob/master/**/*",
         "https://github.com/dotnet/machinelearning/blob/release/**/*",
+        "https://github.com/dotnet/roslyn/blob/dev/**/*",
+        "https://github.com/dotnet/roslyn/blob/features/**/*",
+        "https://github.com/dotnet/roslyn/blob/master**/*",
```
Comment (dagood) [inline Maestro/subscriptions.json]: Maybe you're not done, but noticed:

```suggestion
        "https://github.com/dotnet/roslyn/blob/master/**/*",
```

**Pull request merged**